workers/unity/Packages/com.improbable.gdk.transformsynchronization/Components/BufferedTransform.cs

```diff
@@ -3,7 +3,6 @@
 
 namespace Improbable.Gdk.TransformSynchronization
 {
-    [InternalBufferCapacity(TransformSynchronizationConfig.MaxLoadMatchedBufferSize)]
```

Comment:
It could still be useful to give this a decent default value as it denotes the initial capacity

Reply:
We don't know a static default value. It could be 1, it could be 20.